package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "party3lakefak",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node src/server/server.js",
+    "dev": "nodemon src/server/server.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint ./src",
+    "lint-fix": "eslint --fix ./src",
+    "eslint-init": "eslint --init ./src"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FACK1/Party3laKefak.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FACK1/Party3laKefak/issues"
+  },
+  "homepage": "https://github.com/FACK1/Party3laKefak#readme",
+  "dependencies": {
+    "express": "^4.16.4",
+    "express-handlebars": "^3.0.0",
+    "lint": "^1.1.2"
+  },
+  "devDependencies": {
```

Comment:
you should install "eslint-plugin-import, eslint-config-airbnb-base, eslint" modules